src/server/scripts/Northrend/Ulduar/Ulduar/boss_mimiron.cpp

```diff
@@ -991,7 +990,7 @@ class boss_vx_001 : public CreatureScript
                         me->SetImmuneToPC(false);
                         me->RemoveAurasDueToSpell(SPELL_FREEZE_ANIM);
                         me->SetUInt32Value(UNIT_NPC_EMOTESTATE, EMOTE_ONESHOT_NONE); // Remove emotestate.
-                        //me->SetUInt32Value(UNIT_FIELD_BYTES_1, UNIT_BYTES_1_OFFSET_ANIM_TIER, UNIT_BYTE1_FLAG_ALWAYS_STAND | UNIT_BYTE1_FLAG_HOVER); Blizzard handles hover animation like this it seems.
+                        //me->SetHover(true); // Blizzard handles hover animation like this it seems.
```

Comment:
should this still be commented?

Reply:
we have hover implemented now, but mimiron needs a rewrite xd